Tools/ssl/multissltests.py

```diff
@@ -41,15 +41,14 @@
 log = logging.getLogger("multissl")
 
 OPENSSL_OLD_VERSIONS = [
-     "0.9.8zc",
-     "0.9.8zh",
      "1.0.1u",
+     "1.0.2o",
 ]
 
 OPENSSL_RECENT_VERSIONS = [
-     "1.0.2",
-     "1.0.2l",
-     "1.1.0f",
+     "1.0.2p",
+     "1.1.0i",
+#     "1.1.1",
```

Comment:
is it skipped on purpose?

Reply:
Yes, with this PR, python2 is not yet completely compatible with openssl 1.1.1, it will be with the follow-up PR's.

Reply:
Ok. I preferred to ask.